extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeCommentsExtractorTest.java

```diff
@@ -328,5 +331,17 @@ public void testGetCommentsFirstReplies() throws IOException, ExtractionExceptio
             assertEquals("First", replies.getItems().get(0).getCommentText(),
                     "First reply comment did not match");
         }
+
+        @Test
+        public void testGetCommentsReplyCount() throws IOException, ExtractionException {
+            final InfoItemsPage<CommentsInfoItem> comments = extractor.getInitialPage();
+
+            DefaultTests.defaultTestListOfItems(YouTube, comments.getItems(), comments.getErrors());
+
+            CommentsInfoItem firstComment = comments.getItems().get(0);
```

Comment:
Use `final` here, like it is required in the normal code.

Reply:
I had copy from [here](https://github.com/TeamNewPipe/NewPipeExtractor/blob/197d8876fcbc8126fb74d1c45563f1fe3bef0bcd/extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeCommentsExtractorTest.java#L325).